code/controllers/subsystem/polling.dm

```diff
@@ -184,7 +186,11 @@ SUBSYSTEM_DEF(polling)
 	finishing_poll.trim_candidates()
 	log_game("Candidate poll [finishing_poll.role ? "for [finishing_poll.role]" : "\"[finishing_poll.question]\""] finished. [length_pre_trim] players signed up, [length(finishing_poll.signed_up)] after trimming")
 	finishing_poll.finished = TRUE
-	finishing_poll.alert_button.update_stacks_overlay()
+	if(!(finishing_poll.linked_alert_buttons.len == 0))
+		for(var/atom/movable/screen/alert/poll_alert/linked_button as anything in finishing_poll.linked_alert_buttons)
+			if(QDELETED(linked_button))
+				continue
+			linked_button.update_stacks_overlay()
```

Comment:
1. Don't do `(!(length == 0))`, do `(length != 0)`
2. Don't do `(length != 0)`, you can just do `(length)`
3. You don't need to check length before iterating over a list, if there's nothing in the list, it just won't do anything

```suggestion
	for(var/atom/movable/screen/alert/poll_alert/linked_button as anything in finishing_poll.linked_alert_buttons)
		if(QDELETED(linked_button))
			continue
		linked_button.update_stacks_overlay()
```